python/tests/generic_alias_1.py

```diff
@@ -1,18 +1,22 @@
-from typing import Literal, TypedDict
+from typing import Literal, TypedDict, TypeVar, Generic
+from typing_extensions import TypeAliasType
 
 from typechat import python_type_to_typescript_schema
 
-class First[T](TypedDict):
+T = TypeVar("T", covariant=True)
+
+
+class First(Generic[T], TypedDict):
```

Comment:
Currently TypeChat only understands generics using 3.12 syntax. It doesn't consider types that extend `Generic[...]` or generic protocols as adding type parameters. So I think we either find a way to make some of these tests only run in 3.12, or temporarily disable them.

Reply:
I updated the TypeChat code to read type parameters for the classes that extend Generic[...] from the __parameters__ field. With this, the generic_alias_1.py test now outputs type info correctly:
```typescript

// Entry point is: 'FirstOrSecond'
// TypeScript Schema:

type FirstOrSecond<T> = First<T> | Second<T>

interface Second<T> {
    kind: "second";
    "second_attr": T;
}

interface First<T> {
    kind: "first";
    "first_attr": T;
}
```